Tests/AlicerceTests/AutoLayout/ConstraintGroupToggleTestCase.swift

```diff
@@ -0,0 +1,193 @@
+import XCTest
+@testable import Alicerce
+
+
+
+class ConstraintGroupToggleTestCase: BaseConstrainableProxyTestCase {
+
+    private var constraintGroupToggle: ConstraintGroupToggle<TestConstraintGroupKey>!
+
+    private var constraintGroup1: ConstraintGroup!
+    private var constraintGroup2: ConstraintGroup!
+    private var constraintGroup3: ConstraintGroup!
+
+    private var constraint1: NSLayoutConstraint!
+    private var constraint2: NSLayoutConstraint!
+    private var constraint3: NSLayoutConstraint!
+
+    override func setUp() {
+        super.setUp()
+
+        constraintGroup1 = constrain(self.view0, self.view1, activate: false) { view0, view1 in
```

Comment:
maybe we should test a scenario where we pass in the groups all activated, and expect it to deactivate them? 🤔